src/Razor/test/Microsoft.VisualStudio.LiveShare.Razor.Test/SerializationTest.cs

```diff
@@ -22,7 +22,7 @@ public void ProjectSnapshotHandleProxy_RoundTripsProperly()
                 TagHelperDescriptorBuilder.Create("TestTagHelper", "TestAssembly").Build(),
                 TagHelperDescriptorBuilder.Create("TestTagHelper2", "TestAssembly2").Build(),
             };
-            var projectWorkspaceState = new ProjectWorkspaceState(tagHelpers);
+            var projectWorkspaceState = new ProjectWorkspaceState(tagHelpers, default);
```

Comment:
Can we add an integration tests for the E2E of using this at build-time?

Reply:
Absolutely, this is just a design PR 😄